asserts/pool.go

```diff
@@ -273,6 +362,52 @@ func (p *Pool) phase(ph poolPhase) error {
 	return nil
 }
 
+// AddSequenceToUpdate adds the assertion referenced by toUpdate and all its
+// prerequisites to the Pool as unresolved and as required by the
+// given group. It is assumed that the assertion is currently in the
+// ground database of the Pool, otherwise this will error.
+// The current revisions of the assertion and its prerequisites will
+// be recorded and only higher revisions will then resolve them,
+// otherwise if ultimately unresolved they will be assumed to still be
+// at their current ones. If toUpdate is pinned, then it will be resolved
+// to the highest revision with same sequence point (toUpdate.Sequence).
+func (p *Pool) AddSequenceToUpdate(toUpdate *AtSequence, group string) error {
+	if err := p.phase(poolPhaseAddUnresolved); err != nil {
+		return err
+	}
+	if toUpdate.Sequence <= 0 {
+		return fmt.Errorf("internal error: sequence to update must have a sequence number set")
+	}
```

Comment:
this needs to check  the revision as well, and actually also try to find the assertion in the ground db of the pool. I'm to chat about the details of that. It can also be done in a follow up.

Reply:
This basically needs to use a combination of FindSequence and then Fetcher/Fetcher.Save to do the same as AddToUpdate.

Reply:
mmh, I was probably confused yesterday,  if we have an actual assertion and we know sequence,  Fetcher.Fetch would work here as well